src/main/java/org/blacksmith/Ai.java

```diff
@@ -1,14 +1,14 @@
 package org.blacksmith;
 
+import org.blacksmith.entitytypes.CollisionEntity;
```

Comment:
currently, the AI isn't really managed by the ECS.  I made these changes so that I could start hooking up stuff in the ECM.  I moved a lot of the logic to the Entity system, so that we can add new enemy types easier, and we can move the management out of the Game object